pkgs/tools/misc/glob/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "glob";
```

Comment:
we might want to name it node-glob [like most distros](https://repology.org/project/node:glob/versions)